cmds/core/cmp/cmp.go

```diff
@@ -70,53 +70,50 @@ func openFile(name string) (*os.File, error) {
 }
 
 // cmp is defined to fail with exit code 2
-func failf(format string, a ...interface{}) {
-	fmt.Fprintf(os.Stderr, format, a...)
-	os.Exit(2)
+func failf(w io.Writer, format string, a ...interface{}) int {
```

Comment:
could this not return a syscall.Error with value 2? i.e. the signature is fail(...) error and you return
return syscall.Errno(2)

what is 2 anyway?

Reply:
There is a comment that says that cmp is defined to fail with exit code 2